crates/subspace-gateway/src/commands/run/dsn.rs

```diff
@@ -70,12 +70,12 @@ pub async fn configure_network(
     let farmer_app_info = node_client
         .farmer_app_info()
         .await
-        .map_err(|error| anyhow!("Failed to get farmer app info: {error}"))?;
+        .map_err(|error| anyhow!("Failed to get gateway app info: {error}"))?;
 
     // Fall back to the node's bootstrap nodes.
     if bootstrap_nodes.is_empty() {
         debug!(
-            dsn_bootstrap_nodes = ?farmer_app_info.dsn_bootstrap_nodes,
+            bootstrap_nodes = ?farmer_app_info.dsn_bootstrap_nodes,
```

Comment:
I intentionally left it as `dsn_bootstrap_nodes` because it came from the node to distinguish from explicit option